WPFHospitalEditor/SearchResultDialog.xaml.cs

```diff
@@ -0,0 +1,219 @@
+                }
+                else
+                {
+                    createRowData(mapObject);
+                }
+            }
+        }
+
+        
+
+        private int createRowData(MapObject mapObject)
+        {
+            addLabels(mapObject);
+
+            addAdvancedSearchButton();
+
+            addSeparator();
+
+            firstContentRowNumber++;
+
+            return firstContentRowNumber;
+        }
+
+        private void addLabels(MapObject mapObject)
+        {
+            for(int i = 1; i <= 3; i++)
+            {
+                Label label = new Label();
+
+                adjustLabelProperties(mapObject, label, i);
+
+                Grid.SetRow(label, firstContentRowNumber);
+                Grid.SetColumn(label, i);
+
+                DynamicGrid.Children.Add(label);
+            }
+        }
+
+        private void adjustLabelProperties(MapObject mapObject, Label label, int i)
+        {
+            switch(i)
+            {
+                case 1:
+                    {
+                        label.Content = mapObject.Name;
+                        label.HorizontalAlignment = HorizontalAlignment.Center;
+                        label.VerticalAlignment = VerticalAlignment.Center;
+                    }
+                    break;
+                case 2:
+                    {
+                        label.Content = Building.findBuilding(mapObject);
+                        label.HorizontalAlignment = HorizontalAlignment.Center;
+                        label.VerticalAlignment = VerticalAlignment.Center;
+                    }
+                    break;
+                case 3:
+                    {
+                        label.Content = Building.findFloor(mapObject);
+                        label.HorizontalAlignment = HorizontalAlignment.Center;
+                        label.VerticalAlignment = VerticalAlignment.Center;
+                    }
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        private void addNameLabel(MapObject mapObject)
+        {
+            Label name = new Label();
+
+            adjustNameLabelProperties(mapObject, name);
+
+            Grid.SetRow(name, firstContentRowNumber);
+            Grid.SetColumn(name, 1);
+
+            DynamicGrid.Children.Add(name);
+        }
+
+        private static void adjustNameLabelProperties(MapObject mapObject, Label name)
+        {
+            name.Content = mapObject.Name;
+            name.HorizontalAlignment = HorizontalAlignment.Center;
+            name.VerticalAlignment = VerticalAlignment.Center;
+        }
+
+        private void addAdvancedSearchButton()
+        {
+            Button advancedSearch = new Button();
+
+            adjustAdvancedSearchButtonProperties(advancedSearch);
+
+            advanceSearchButtons.Add(advancedSearch);
+
+            //advancedSearch.Click += Advanced_Search_Click(mapObject, i);
```

Comment:
unless it's absolutely necessary, avoid leaving comments, especially commented code

Reply:
Those comments are related to next feature, but I will delete them.